doc/man3/BN_generate_prime.pod

```diff
@@ -101,7 +101,14 @@ If B<do_trial_division == 0>, this test is skipped.
 Both BN_is_prime_ex() and BN_is_prime_fasttest_ex() perform a Miller-Rabin
 probabilistic primality test with B<nchecks> iterations. If
 B<nchecks == BN_prime_checks>, a number of iterations is used that
-yields a false positive rate of at most 2^-80 for random input.
+yields a false positive rate of at most 2^-64 for random input.
+The error rate goes down for bigger primes.
+
+When the source of the prime is not random or not trusted, the number
+of checks should be set to the security level in bits you want to reach
+divided by 2 and rounded up.
+For instance if you want to reach the 128 bit level, B<nchecks> should be set
```

Comment:
Isn't there a comma missing after 'For instance'?

Reply:
Alternative formulation (just a suggestion, not a request): 

> When the source of the prime is not random or not trusted, the number
of checks needs to be much higher to reach the same level of assurance:
It should equal half of the targeted security level in bits (rounded up to the
next integer if necessary). For instance, to reach the 128 bit security level,
B\<nchecks\> should be set to 64.